nativescript-angular/router/page-router-outlet.ts

```diff
@@ -102,6 +102,8 @@ function routeToString(activatedRoute: ActivatedRoute | ActivatedRouteSnapshot):
     return activatedRoute.pathFromRoot.join("->");
 }
 
+const routeTransitionWarning = "This could be due to route transition timing and could be ignored.";
```

Comment:
@vakrilov shoot so sorry this should be deleted as I don’t believe it ended up being used - not in front of computer but could fix up tomorrow or if you just want to remove this line just wanted to flag it before it went in.